### PR TITLE
chore: fix maven + docker version naming

### PR DESCRIPTION
**Pull request by lct45**
### Description 
change the naming so it's `0.v.0` instead of `0.v.0-1` for maven and docker
### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: build-packages.sh [hunk 1]
```diff
@@ -110,8 +110,8 @@ export DEBEMAIL="Confluent Packaging <packages@confluent.io>"
 xmlstarlet ed --inplace -P --update "/_:project/_:parent/_:version" --value "${UPSTREAM_VERSION}" pom.xml
 
 # Maven provides some helpfull commands for setting versions/properties
-mvn --batch-mode versions:set          -DgenerateBackupPoms=false "-DnewVersion=${FULL_VERSION}"
-mvn --batch-mode versions:set-property -DgenerateBackupPoms=false "-DnewVersion=${FULL_VERSION}" -Dproperty=io.confluent.ksql.version 
+mvn --batch-mode versions:set          -DgenerateBackupPoms=false "-DnewVersion=${VERSION}"
```
Comment (lct45) [inline build-packages.sh:113]: This I'm not sure about - I don't know how these maven commands are used

**Comment by andrewegel** [comment]
I think you'll also need to update:

```
fakeroot make PACKAGE_TYPE=rpm "VERSION=${FULL_VERSION}" "RPM_VERSION=${VERSION}" "REVISION=${RELEASE}" -f debian/Makefile rpm
```

to 

```
fakeroot make PACKAGE_TYPE=rpm "VERSION=${VERSION}" "RPM_VERSION=${VERSION}" "REVISION=${RELEASE}" -f debian/Makefile rpm
```

and

```
dch --newversion "${FULL_VERSION}" "Release version ${FULL_VERSION}"
```

to 

```
dch --newversion "${VERSION}" "Release version ${VERSION}"
```

Getting a PR build is going to give us confidence in the change.

**Comment by lct45** [comment]
@andrewegel made those updates, it looks like it's building okay. Is there a different way to confirm that this will work ok?

**Comment by andrewegel** [comment]
> @andrewegel made those updates, it looks like it's building okay. Is there a different way to confirm that this will work ok?

The `Semantic Pull Request ` check passed yes, but there is no Jenkins check that passed - Jenkins hasn't picked up this PR build yet due to hitting GitHub API rate limits. It will be picked up later today, from there we should adjust accordingly.

File: build-packages.sh (hunk 1 above)
Comment (vcrfxia) [reply build-packages.sh:113]: This change is correct. We want the maven artifacts to be versioned as just `0.17.0`, not `0.17.0-1`.

**Review by vcrfxia — COMMENTED** [review]
I'm confused. Where is `FULL_VERSION` still being used after these changes? This PR looks like it's using `VERSION` (rather than `FULL_VERSION`) for building everything now: deb, rpm, tgz, maven artifacts, and docker.

**Comment by lct45** [comment]
I see it used as the work branch: `work_branch="debian-${FULL_VERSION}"` but nowhere else... the build also failed so maybe I replaced some too many

File: build-packages.sh [hunk 2]
```diff
@@ -110,20 +110,20 @@ export DEBEMAIL="Confluent Packaging <packages@confluent.io>"
...
+mvn --batch-mode versions:set-property -DgenerateBackupPoms=false "-DnewVersion=${VERSION}" -Dproperty=io.confluent.ksql.version
 mvn --batch-mode versions:set-property -DgenerateBackupPoms=false "-DnewVersion=${UPSTREAM_VERSION}" -Dproperty=io.confluent.schema-registry.version
 
 # Set version for Debian Package
-dch --newversion "${FULL_VERSION}" "Release version ${FULL_VERSION}"
+dch --newversion "${VERSION}" "Release version ${VERSION}"
```
Comment (andrewegel) [inline build-packages.sh:118]:
```suggestion
dch --newversion "${FULL_VERSION}" "Release version ${FULL_VERSION}"
```

I'm seeing this error in the PR build:

```
14:19:13  gbp:info: confluent-ksqldb_None.orig.tar.gz does not exist, creating from 'debian-0.18.0'
```

Which causes issues later on, in my local docker experiments changing this back to FULL_VERSION gets it working.

**Review by andrewegel — APPROVED** [review]
I see the PR build has passed.

**Pull request merged**